docs/versioned_docs/version-2.11.0/introduction/community.md

```diff
@@ -1,6 +1,10 @@
 ---
 id: community
 title: OpenEBS Community
+keywords:
+  - Community
+  - OpenEBS Community
```

Comment:
Keywords word case are different, for some both the starting letters are capital and some only the first word starts with a capital, would be better if we follow one approach throughout.